lib/nodeMap.js

```diff
@@ -64,7 +64,7 @@ api.createNodeMap = (input, graphs, graph, issuer, name, list) => {
   // add values to list
   if(graphTypes.isValue(input)) {
     if('@type' in input) {
-      let type = input['@type'];
+      const type = input['@type'];
```

Comment:
A few lines down `type` can change so needs to be a `let` here.  Maybe test coverage doesn't hit that conditional.